pkg/cli/cmds/server.go

```diff
@@ -508,6 +509,11 @@ var ServerFlags = []cli.Flag{
 		Usage:       "(experimental) Run rootless",
 		Destination: &ServerConfig.Rootless,
 	},
+	cli.BoolFlag{
+		Name:        "prefer-bundled-bin",
+		Usage:       "(experimental) Prefer bundled userspace binaries over host binaries",
+		Destination: &ServerConfig.AuxTools,
```

Comment:
If we're not going to use the `AuxTools` field (or whatever we call it) anywhere, you can just drop it from the struct and don't bind the flag to a destination. It'll just be available as `cli.String("prefer-bundled-bin)`.